### ISSUE TITLE
Page translation request: outdated or missing Portuguese Brazilian pages

### ISSUE THREAD

**Issue by sebastiaanspeck** [tldr-pages/tldr#13575]
Right now (2024-09-02) there are:

- 139 missing alias page(s) in check-pages.pt_BR/missing-pt_BR-alias-pages.txt.
- 147 missing TLDR page(s) in check-pages.pt_BR/missing-tldr-pt_BR-commands.txt.
- 75 outdated page(s) based on number of commands in check-pages.pt_BR/outdated-pt_BR-pages-based-on-command-count.txt.
- 40 outdated page(s) based on the commands itself in check-pages.pt_BR/outdated-pt_BR-pages-based-on-command-contents.txt.

This is according to https://github.com/tldr-pages/tldr-maintenance. I see we are constantly adding new pages, which is great, but if we keep this stack above building up, it won’t be feasible to fix these issues anymore. 
Therefore I would like to open this request to call Portuguese Brazilian maintainers to keep the Portuguese Brazilian pages up-to-date, besides adding new pages.

@isaacvicente @vitorhcl

> [!NOTE]  
> This issue is for Portuguese Brazilian only, but I could open way more issues for all other CODEOWNER-maintained languages. I want to mention again how useful https://github.com/tldr-pages/tldr-maintenance is to me. I can check very quickly if I have to do any work on updating the Dutch pages e.g., or if we have to run set-more-info-link.py e.g.

**Comment by renie** [comment]
Sure, @sebastiaanspeck . I will have a look at them.

**Comment by renie** [comment]
[first PR submitted](https://github.com/tldr-pages/tldr/pull/13800) for all the outdated linux commands based on the commands itself.

**Comment by renie** [comment]
[second PR submitted](https://github.com/tldr-pages/tldr/pull/13830) for all the outdated linux commands based on **the number of commands**.

**Comment by rffontenelle** [comment]
> For your information this issue has outdated numbers, please refer to https://github.com/tldr-pages/tldr-maintenance/issues/110 which is an up-to-date issue and will also be updated if you would resolve any outdated page.

Thanks. I've spotted the aforementioned issue before. Congrats on the this very organized system, it was very easy to spot the contribution needed and how to contribute.

**Comment by renie** [comment]
Hey @rffontenelle ! It is very nice to see someone else interested on helping with those translations.
About your questions on where to start. I am working on a PR for `common` and I have finished the `linux` part. I think it would be nice if you follow your own suggestion of starting with `windows` and `osx`, maybe even `android` ones.
And if you finish all of those before I finish the `common` ones, we can find an approach to work together on those. What do you think?

**Comment by rffontenelle** [comment]
@renie thanks for letting me know. Ok, windows and osx first, then we coordinate.

**Comment by renie** [comment]
It is very good to see those lists getting smaller 😌

**Comment by rafaelgpaves** [comment]
Hi, I'd also like to contribute to translation. I see you're already working on `windows`, `osx` and `common`. Is it ok if I start working on missing alias pages in `freebsd` or maybe something outdated in `common` like mysql?

**Comment by renie** [comment]
Sure, @rafaelgpaves ! `freebsd` ones seems to be a good place to start now. Once finished we can talk about `common`. Sounds good to you?

**Comment by rafaelgpaves** [comment]
Sounds good! I'll start working on `freebsd` then.

**Comment by johnnywell** [comment]
Hello! I'd also like to contribute to to the translation. Submititted my first PR: https://github.com/tldr-pages/tldr/pull/14401 that fixes a broken a link and adds a 'dreprecated' warning.

**Comment by renie** [comment]
> Hello! I'd also like to contribute to the translation. Submititted my first PR: [#14401](https://github.com/tldr-pages/tldr/pull/14401) that fixes a broken a link and adds a 'deprecated' warning.

Thanks, @johnnywell !

**Comment by CleyJosRM** [comment]
Hi! I was looking at the project and my group of friends and I would be happy to contribute! We're thinking of starting by translating some Linux commands into Portuguese.

**Comment by sebastiaanspeck** [comment]
> Hi! I was looking at the project and my group of friends and I would be happy to contribute! We're thinking of starting by translating some Linux commands into Portuguese.

Feel free to contribute by opening PRs.

**Comment by CleyJosRM** [comment]
Sorry, maybe I'm misunderstanding something, but shouldn't the PR be merged immediately after obtaining both approvals?

**Comment by sebastiaanspeck** [comment]
> Sorry, maybe I'm misunderstanding something, but shouldn't the PR be merged immediately after obtaining both approvals?

No, some languages have codeowners so we wait for a few days till they review it. If they don't, we merge it. Please read our contributing guide for further clarification.

**Comment by Managor** [comment]
Furthermore, if you contribute enough, you can become a maintainer and a codeowner yourself to speed up the review process
